UKHomeOffice/sgmr-service
# Fixes menu button

## AC
Fixes menu button. Menu button doesn't show on desktop, shows correctly on mobile.


----

## To test
Check menu button displays on mobile and doesn't display on desktop. Check arrow rotates on expand/collapse

----

## Notes
N/A
